mqttmpd.c

```diff
@@ -584,6 +584,12 @@ int main(int argc, char *argv[])
 		break;
 	case 'p':
 		mpd_host = optarg;
+		str = strrchr(optarg, ':');
+		if (str > mpd_host && *(str-1) != ']') {
```

Comment:
Why `*(str-1) != ']'`?
You hit the ipv6 closely, but I do not understand how your test works in such case (like `-p [::1]:6601` or so)

Reply:
stupid question of mine, you did the same as for MQTT :-)